.travis.yml

```diff
@@ -10,12 +10,11 @@ cache:
     - node_modules
 
 before_install:
-  - export PATH=/usr/local/phantomjs-2.0.0/bin:$PATH
-  - "npm config set spin false"
-  - "npm install -g npm@^2"
+  - npm config set spin false
+  - npm install -g bower
+  - npm install phantomjs-prebuilt
 
 install:
-  - npm install -g bower
   - npm install
   - bower install
 
```

Comment:
Wait, what - do we use Travis? I didn't know that. We have no tests whatsoever at the moment, so.. was this autogenerated or sth?